### PR TITLE
CB-7517 Verify tags automatically within the testing framework

### PR DESCRIPTION
**Pull request by Bajzathd**

### CODE REVIEW

File: core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/response/tags/TagsV4Response.java [hunk 1]
```diff
@@ -42,4 +44,12 @@ public void setUserDefined(Map<String, String> userDefined) {
     public void setDefaults(Map<String, String> defaults) {
         this.defaults = defaults;
     }
+
+    @Override
+    public String getTagValue(String key) {
+        return Optional.ofNullable(application.get(key))
+                .or(() -> Optional.ofNullable(userDefined.get(key)))
```
Comment (bergerdenes) [inline core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/response/tags/TagsV4Response.java:51]: question: which has a higher preference, the user-defined tags, or application tags?

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/environment/EnvironmentTestDto.java [hunk 2]
```diff
@@ -42,7 +42,6 @@
 import com.sequenceiq.it.cloudbreak.dto.credential.CredentialTestDto;
 import com.sequenceiq.it.cloudbreak.dto.telemetry.TelemetryTestDto;
 import com.sequenceiq.it.cloudbreak.search.Searchable;
-import com.sequenceiq.it.util.TagAdderUtil;
 
 @Prototype
 public class EnvironmentTestDto
```
Comment (bergerdenes) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/environment/EnvironmentTestDto.java:47]: this is not inherited from AbstractTestDto!

Comment (Bajzathd) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/environment/EnvironmentTestDto.java:47]: it is a bit more abstract, but inherited
image

Comment (bergerdenes) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/environment/EnvironmentTestDto.java:47]: I didn't see the `extends` line below, sorry.

**Comment by Bajzathd** [comment]
retest this please

**Pull request merged**